README.rst

```diff
@@ -127,11 +127,27 @@ and run the following from the root directory of the project::
 
     $ pipenv install --dev --three
 
-This will create a virtualenv with all packages and dev-packages installed. Now you can, for example,
-run all tests via::
+This will create a virtualenv with all packages and dev-packages installed.
+
+
+Tests
+-----
+All CI tests run against an actual GCS bucket provided by `Othoz GmbH <http://othoz.com/>`__. In order to run the tests against your own bucket,
+make sure to set up a `Service Account <Othoz GmbH https://cloud.google.com/iam/docs/service-accounts>`__ with all necessary permissions:
+
+- storage.buckets.get
+- storage.objects.get
+- storage.objects.list
+- storage.objects.create
+- storage.objects.update
+- storage.objects.delete
+
```

Comment:
We could even add to that explanation by saying that the best matching predefined Role is "Storage Admin" assigned to the specific bucket (in order to avoid having to create custom Roles).